wrappers/services_test.go

```diff
@@ -163,6 +163,291 @@ WantedBy=multi-user.target
 	c.Check(s.sysdLog[1], DeepEquals, []string{"daemon-reload"})
 }
 
+func (s *servicesTestSuite) TestEnsureSnapServicesAdds(c *C) {
+	info := snaptest.MockSnap(c, packageHello, &snap.SideInfo{Revision: snap.R(12)})
+	svcFile := filepath.Join(s.tempdir, "/etc/systemd/system/snap.hello-snap.svc1.service")
+
+	m := map[*snap.Info]*wrappers.AddSnapServicesOptions{
+		info: nil,
+	}
+
+	modified, err := wrappers.EnsureSnapServices(m, progress.Null)
+	c.Assert(err, IsNil)
+	c.Check(s.sysdLog, DeepEquals, [][]string{
+		{"daemon-reload"},
+	})
+	c.Assert(modified, HasLen, 1)
+	for modifiedSn, modifiedApps := range modified {
+		// don't try a DeepEquals on this, it could be a recursive data
+		// structure which go-check doesn't handle very well
+		c.Assert(modifiedSn.SnapName(), Equals, info.SnapName())
+		c.Assert(modifiedApps, HasLen, 1)
+		c.Assert(modifiedApps, DeepEquals, []*snap.AppInfo{info.Apps["svc1"]})
+	}
+
+	content, err := ioutil.ReadFile(svcFile)
```

Comment:
(nitpick^2) We could use `testutil.FileEquals` here (and in the other tests) which is slightly more compact.

Reply:
sure I did this